server/apps/research/serializers/article_serializer.py

```
@@ -120,5 +190,6 @@
                 instance.related_articles.set(related_articles)
 
             return instance
-        except Exception as e:          
-            raise serializers.ValidationError(f"Error updating article: {str(e)}")
+        except Exception as e:
+            logging.error(f"Error updating article: {str(e)}")
+            raise serializers.ValidationError("An error occurred while updating the article.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)